dockerfile_parse/parser.py

```diff
@@ -149,9 +153,20 @@ def _open_dockerfile(self, mode):
             yield self.fileobj
             self.fileobj.seek(0)
         else:
-            with open(self.dockerfile_path, mode) as dockerfile:
+            with self.dockerfile.open(mode) as dockerfile:
                 yield dockerfile
 
+    @property
+    def dockerfile_path(self):
+        """
+        :return: Path to the Dockerfile as a string, or None if using fileobj
+        """
+        return str(self.dockerfile) if self.dockerfile else None
+    
```

Comment:
flake8 is saying there's extra whitespace on this line (165)

Reply:
Ah sorry, missed that somehow.